include/functions.h

```diff
@@ -378,8 +378,8 @@ void Actor_MoveForward(Actor* actor);
 void func_8002D908(Actor* actor);
 void func_8002D97C(Actor* actor);
 void func_8002D9A4(Actor* actor, f32 arg1);
-s16 Actor_WorldYawTowardActor(Actor* actorA, Actor* actorB);
-s16 Actor_WorldYawTowardPoint(Actor* actor, Vec3f* refPoint);
+s16 Actor_WorldYawTowardActor(Actor* actorOrigin, Actor* actorTarget);
+s16 Actor_WorldYawTowardPoint(Actor* actorOrigin, Vec3f* point);
```

Comment:
`originActor` and `targetActor` maybe? That reads better to me at least, and it feels like the natural order.

Also I think those and the other functions below should follow a consistent pattern, so changing the yaw ones on their own doesn't feel right

Reply:
I think actorOrigin/actorTarget is derived from the previous names actorA/actorB but I don't even think "actor" is even needed at all, thoughts?

The overwhelmingly conflicted feedback about touching these yaw functions made me unwilling to touch the pitch ones but I can do it too. I'll do it eventually

Reply:
I was going to comment that it seems unneeded, since that information is in the type. but I didnt care too much to actually start a discussion on it.
If we do keep actor then I think it makes sense with romans suggestion

Reply:
Agreed with fig. `Actor` does seem a little redundant (Even `actorA` and `actorB` could just be `a` and `b`, since that is they way the `Vec3f` versions work).